<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel Organizer | Panel Administracyjny</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/img/icon-dashboard.png</normaloff>:/images/img/icon-dashboard.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBoxDatabase">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Baza Hotel</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelClientList">
         <property name="text">
          <string>Lista klientów</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetClients"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxManage">
      <property name="title">
       <string>Zarządzanie Hotel</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBoxClient">
         <property name="title">
          <string>Klient</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelClientName">
            <property name="text">
             <string>Imię</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditClientName">
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelClientSurname">
            <property name="text">
             <string>Nazwisko</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditClientSurname">
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelClientPesel">
            <property name="text">
             <string>Pesel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditClientPesel">
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>11</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelClientDate">
            <property name="text">
             <string>Data rezerwacji</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelClientDays">
            <property name="text">
             <string>Czas rezerwacji (ilość dni)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="spinBoxClientRoomNumber">
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelClientRoom">
            <property name="text">
             <string>Nr pokoju</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelClientPeople">
            <property name="text">
             <string>Liczba osób</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="spinBoxClientPeople">
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateEdit" name="dateEditClientDate">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2017</year>
              <month>2</month>
              <day>11</day>
             </datetime>
            </property>
            <property name="minimumDateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2017</year>
              <month>2</month>
              <day>11</day>
             </datetime>
            </property>
            <property name="displayFormat">
             <string>dd-MM-yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBoxClientDays">
            <property name="maximum">
             <number>365</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxBasicOperations">
         <property name="title">
          <string>Podstawowe operacje</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonClientAdd">
            <property name="text">
             <string>Dodaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonClientSearch">
            <property name="text">
             <string>Szukaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonClientEdit">
            <property name="text">
             <string>Edytuj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonClientDelete">
            <property name="text">
             <string>Usuń</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxOtherOperations">
         <property name="title">
          <string>Dodatkowe operacje</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButtonManageRooms">
            <property name="text">
             <string>Zarządzaj pokojami</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonResetForm">
            <property name="text">
             <string>Resetuj formularz</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxInfo">
         <property name="title">
          <string>Informacje o hotelu</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelInfoClients">
            <property name="text">
             <string>Liczba klientów</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelInfoClientsNumber">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonResetForm</sender>
   <signal>clicked()</signal>
   <receiver>lineEditClientName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>991</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>750</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonResetForm</sender>
   <signal>clicked()</signal>
   <receiver>lineEditClientSurname</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>991</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonResetForm</sender>
   <signal>clicked()</signal>
   <receiver>lineEditClientPesel</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>991</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>874</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonResetForm</sender>
   <signal>clicked()</signal>
   <receiver>spinBoxClientPeople</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>991</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonResetForm</sender>
   <signal>clicked()</signal>
   <receiver>spinBoxClientRoomNumber</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>991</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonResetForm</sender>
   <signal>clicked()</signal>
   <receiver>dateEditClientDate</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>963</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>961</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonResetForm</sender>
   <signal>clicked()</signal>
   <receiver>spinBoxClientDays</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>925</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
